docs/developer-guide/release-process-and-cadence.md

```diff
@@ -88,4 +88,4 @@ These evaluations vary from dependency to dependencies.
 Dependencies are also scheduled for removal if the project has been deprecated or if the project is no longer maintained.
 
 CVEs in dependencies will be patched for all supported versions if the CVE is applicable and is assessed by Snyk to be
-of high or critical severity. Automation generates a [new Snyk scan weekly](../snyk).
+of high or critical severity. Automation generates a [new Snyk scan weekly](../../snyk).
```

Comment:
About the only signifacant change. The rest is cosmetics.

Reply:
Doesn't this break the markdown? Shouldn't it be `[new Snyk scan weekly](../snyk/index.md)`?

Reply:
It does not, I have verified using `make serve-docs` that this fixes the problem.

Page with the link https://argo-cd.readthedocs.io/en/latest/developer-guide/release-process-and-cadence/
Old destination https://argo-cd.readthedocs.io/en/latest/developer-guide/snyk/ (404)
New destination https://argo-cd.readthedocs.io/en/latest/snyk/